install-paperless-ngx.sh

```diff
@@ -346,7 +346,7 @@ read -r -a OCR_LANGUAGES_ARRAY <<< "${_split_langs}"
 	fi
 } > docker-compose.env
 
-sed -i "s/- 8000:8000/- $PORT:8000/g" docker-compose.yml
+sed -i "s/- 8000:8000/- \"$PORT:8000\"/g" docker-compose.yml
```

Comment:
The 8000:8000 portion needs the quotes as well or it won't match and replace in my testing

Reply:
that's odd, i checked my generated yaml file after and it looked correct, but i committed the additional quotes. probably good anyway, since otherwise we might get double double-quotes